src/rabbit_mirror_queue_sync.erl

```diff
@@ -414,3 +416,7 @@ batch_publish_delivered(Batch, MA, BQ, BQS) ->
 
 props(Props) ->
     Props#message_properties{needs_confirming = false}.
+
+run_gc_soon() ->
+    rabbit_core_metrics_gc:run_gc_soon(),
+    rabbit_mgmt_gc:run_gc_soon().
```

Comment:
`rabbit` application can never call anything on `rabbitmq_management` or `rabbitmq_management_agent` as those plugins might or might not be active. The dependency is the other way around.

```
=ERROR REPORT==== 3-Jul-2017::14:07:12 ===
Error in process <0.631.0> on node 'rmq-ct-cluster_tests-1-21000@localhost' with exit value:
{undef,[{rabbit_mgmt_gc,run_gc_soon,[],[]},
        {rabbit_mirror_queue_sync,syncer_loop,3,
                                  [{file,"src/rabbit_mirror_queue_sync.erl"},
                                   {line,266}]}]}
```